packages/gestalt/src/utils/positioningUtils.js

```diff
@@ -160,13 +160,13 @@ export function getPopoverDir({
   const skipNoEdgeCondition =
     isScrollBoundaryContainer && (nonTopEdge || nonBottomEdge) && (nonLeftEdge || nonRightEdge);
 
-  if (!skipNoEdgeCondition && (nonTopEdge || nonBottomEdge)) {
+  if (idealDirection !== 'forceDown' && !skipNoEdgeCondition && (nonTopEdge || nonBottomEdge)) {
```

Comment:
might be possible to wrap this condition on the outside of both statements

```
if(idealDir !=== forceDown){
// do the internal logic
}

Reply:
DONE